packages/oncoprintjs/rules/geneticrules.ts

```diff
@@ -189,7 +191,7 @@ const shapeBank = {
         height: 33.33,
         z: 6,
     },
-    [ShapeId.otherMutationRectangle]: {
+    [ShapeId.otherMutationDriverRectangle]: {
```

Comment:
We should make sure that we don't break any existing Oncoprint functionality with this change. I am not sure if we have tests covering this part.